src/views/v2/month/calendar-body.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * View: Month View - Calendar Body
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/calendar-body.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 4.9.3
```

Comment:
This shouldn't stop us from merging, but it'd be good if we all start setting the TBD version every time we touch a template. Then it will be replaced during the release tasks